### PR TITLE
#615 - rename _from_pem to _crypto_key

### PR DESCRIPTION
**Pull request by NicholasTanz**
Fixes: #615  

### Description of the changes being introduced by the pull request:
* renaming _from_key() method to _crypto_key()

### Please verify and check that the pull request fulfils the following requirements:

- [x] The code follows the [Code Style Guidelines](https://github.com/secure-systems-lab/code-style-guidelines#code-style-guidelines)
- [ ] Tests have been added for the bug fix or new feature
- [ ] Docs have been added for the bug fix or new feature

### CODE REVIEW

**Comment by NicholasTanz** [comment]
Hey, @jku I was wondering if you could take a look at this PR when you got a minute, thanks.

**Review by jku — CHANGES_REQUESTED** [review]
Thanks, looks good. Can you also update the docstring?

File: securesystemslib/signer/_key.py [hunk 1]
```diff
@@ -238,11 +238,9 @@ def from_dict(cls, keyid: str, key_dict: Dict[str, Any]) -> "SSlibKey":
     def to_dict(self) -> Dict[str, Any]:
         return self._to_dict()
 
-    def _from_pem(self) -> "PublicKeyTypes":
+    def _crypto_key(self) -> "PublicKeyTypes":
         """Helper to load public key instance from PEM-formatted keyval.
```
[the remark this answers is not in the document]
Comment (NicholasTanz) [reply securesystemslib/signer/_key.py]: gotcha - just updated it.

**Review by jku — APPROVED** [review]
LGTM. Linter still wants a whitespace tweak, I can fix that

**Comment by jku** [comment]
Thanks for the patch, merged

**Pull request merged**